app/views/playlists/edit_white.html.erb

```diff
@@ -71,7 +71,7 @@
     </p>
     <div class="box">
       <div class="left_column_box_header">
-        <h2><%= @playlist.title %> <span class="playlist_date">(2018)</span></h2>
+        <h2><%= @playlist.title %> <span class="playlist_date">(<%= @playlist.year %>)</span></h2>
```

Comment:
Percy is showing that this opens up a problem when a date isn't present — we get empty parens...We probably want to wrap the whole span in an `if playlist.year.present?`

Reply:
@sudara 

I might argue against having the date here at all, as horizontal space is a premium here for long playlist titles, and the date is clearly in the form at the top of the page?

Reply:
Even better — lets kill it!